hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/StorageImpl.java

```diff
@@ -66,6 +67,11 @@ public void clear(boolean isDuringShutdown) {
     }
 
     @Override
+    public Iterator<R> iterator(boolean failFast) {
```

Comment:
I'm forgotting one thing.. In this case you are ignoring the 'failFast' option.

Does it mean this code is broken or does it mean that the implementation if free to ignore the setting?